js/private/js_binary.bzl

```diff
@@ -604,7 +605,9 @@ js_binary_lib = struct(
         # TODO: on Windows this toolchain is never referenced
         "@bazel_tools//tools/sh:toolchain_type",
         "@rules_nodejs//nodejs:toolchain_type",
-    ],
+    ] + ([
+        "@aspect_bazel_lib//lib:coreutils_toolchain_type",
+    ] if _BAZEL_LIB_VERSION.startswith("2") else []),
```

Comment:
There will be version 3 at some point, probably worth parsing a bit more like digits before first dot